nx.json

```diff
@@ -2,7 +2,11 @@
   "$schema": "node_modules/nx/schemas/nx-schema.json",
   "namedInputs": {
     "default": ["{projectRoot}/**/*"],
-    "prod": ["!{projectRoot}/test/**/*"]
+    "prod": [
+      "{projectRoot}/src/**/*",
+      "{projectRoot}/package.json",
+      "{projectRoot}/tsconfig.json"
```

Comment:
The root tsconfig extends the one from the test folder, but that one is not listed

(yes it's a weird setup, we'll be able to clean it up now that we only support TS 4.9+)

Reply:
The projectRoot here is the package folder. That tsconfig does not extend the one from the test folder right?

Reply:
Ah in that case, good to go